### PR TITLE
Refactor script location

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Move Scripts to Eng/Common Directory

### CODE REVIEW

File: eng/common/pipelines/templates/steps/verify-path-length.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Template for all Python Scripts in this repository
+parameters: 
+  SourceDirectory: ''
+  BasePathLength: 49
+
+steps:
+  - task: PythonScript@0
```
Comment (weshaggard) [inline eng/common/pipelines/templates/steps/verify-path-length.yml:7]: As part of these being moved into eng/common we should also pull out the inline scripts into standalone script files for easier debugging and local running.

File: eng/common/pipelines/templates/steps/replace-relative-links.yml [hunk 2]
```diff
@@ -0,0 +1,217 @@
+parameters: 
+  TargetFolder: ''
+  RootFolder: ''
+  BuildSHA: ''
+  RepoId: ''
+
+steps:
```
Comment (weshaggard) [inline eng/common/pipelines/templates/steps/replace-relative-links.yml:7]: I know @KarishmaGhiya was planning to replace this template so we should figure out if we should wait until then to update usages of this template.

**Comment by weshaggard** [comment]
@chidozieononiwu is this PR just part of moving more things into eng/common so we can remove repository references?

**Comment by chidozieononiwu** [comment]
> @chidozieononiwu is this PR just part of moving more things into eng/common so we can remove repository references?

Right this will enable us get rid of most of the repository resources.

**Pull request merged**